### PR TITLE
Readme and release page for NNIv1

### PR DESCRIPTION
**Pull request by rabbit008**
Readme and release note for v1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ The tool dispatches and runs trial jobs generated by tuning algorithms to search
           <li><a href="docs/en_US/Tuner/BuiltinTuner.md#BOHB">BOHB</a></li>
           <li><a href="docs/en_US/Tuner/BuiltinTuner.md#GPTuner">GP Tuner</a></li>
           </ul>
-          <li><b>Tuner for <a href="docs/en_US/CommunitySharings/NasComparision.md">NAS</a></b></li>
+          <li><b>Tuner for <a href="docs/en_US/AdvancedFeature/GeneralNasInterfaces.md">NAS</a></b></li>
```
Comment (xuehui1991) [inline README.md]: I don't think there needs a link here.

File: docs/en_US/Release_v1.0.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+<br />
+<br />
+
+<p align="center">
+<img src=".././img/release-1-title-2.png" width="100%" />
+</p>
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**Step one**：Start with the [Tutorial Document](./Tutorial/Installation.md),and install NNI v1.0 first.<br>
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**Step two**：Find a " Hello word  example", follow the [Tutorial Documentation](./Tutorial/QuickStart.md) and have a Quick Start. <br>
```
Comment (xuehui1991) [inline docs/en_US/Release_v1.0.md]: Hollo Word? Or Mnist?

File: docs/en_US/Release_v1.0.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+
+<p align="center">
+<img src=".././img/release-1-title-1.png" width="100%" />
+</p>
+
+
+From September, 2018 to September, 2019, We are still moving on …
+
+
+ **Great news!**&nbsp;&nbsp;With the tag of **Scalability** and **Ease of Use** , NNI v1.0 is comming. Based on the various types of [Tuning Algorithms](./Tuner/BuiltinTuner.md),NNI has supported the [Neural Architecture Search](docs/en_US/AdvancedFeature/GeneralNasInterfaces.md),which is an exciting news for algorithmic engineers, besides these, NNI v1.0 has made many improvements in the [platform diversification](./TrainingService/SupportTrainingService.md) and [WebUI simplification](./Tutorial/WebUI.md).NNI has grown into a more intelligent automated machine learning(AutoML) toolkit.
```
Comment (xuehui1991) [inline docs/en_US/Release_v1.0.md]: Seems not contains all v1.0 features.

File: README.md [hunk 4]
```diff
@@ -345,16 +345,28 @@ Before providing your hacks, you can review the [Contributing Instruction](docs/
...
+  * Run [Neural Network Architecture Search](examples/trials/nas_cifar10/README.md) with NNI 
+  * [Automatic Feature Engineering](examples/trials/auto-feature-engineering/README.md) with NNI 
+  * [Hyperparameter Tuning for Matrix Factorization](https://github.com/microsoft/recommenders/blob/master/notebooks/04_model_select_and_optimize/nni_surprise_svd.ipynb) with NNI
+
+* ### **Relevant Articles** ###
+  
+  * [Hyper Parameter Optimization Comparison](docs/en_US/CommunitySharings/HpoComparision.md)
+  * [Neural Architecture Search Comparison](docs/en_US/CommunitySharings/NasComparision.md)
+  * [Parallelizing a Sequential Algorithm TPE](docs/en_US/CommunitySharings/ParallelizingTpeSearch.md)
+  * [Automatically tuning SVD on NNI](docs/en_US/CommunitySharings/RecommendersSvd.md)
```
Comment (xuehui1991) [inline README.md]: consider add "SPTAG" here.

Comment (rabbit008) [reply README.md]: fixed

File: docs/en_US/Release_v1.0.md (hunk 2 above)
Comment (rabbit008) [reply docs/en_US/Release_v1.0.md]: maybe change to "Hello word(Mnist) example"  ?

File: README.md (hunk 1 above)
Comment (rabbit008) [reply README.md]: But some users may be confused about the meaning of “NAS”. there is a simple description in this link

Comment (xuehui1991) [reply README.md]: the "general NAS interface" is not equal to "NAS"

File: docs/en_US/Release_v1.0.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
+
+<p align="center">
+<img src=".././img/release-1-title-1.png" width="100%" />
+</p>
+
+
+From September, 2018 to September, 2019, We are still moving on …
+
+
+ **Great news!**&nbsp;&nbsp;With the tag of **Scalability** and **Ease of Use** , NNI v1.0 is comming. Based on the various types of [Tuning Algorithms](./Tuner/BuiltinTuner.md), NNI has supported the [Neural Architecture Search](docs/en_US/AdvancedFeature/GeneralNasInterfaces.md) and  Auto-Feature generator & selection,which is an exciting news for algorithmic engineers, besides these, NNI v1.0 has made many improvements in the optimization of tuning algorithm,  [WebUI's simplicity and intuition](./Tutorial/WebUI.md) and [platform diversification](./TrainingService/SupportTrainingService.md). NNI has grown into a more intelligent automated machine learning(AutoML) toolkit.
```
Comment (QuanluZhang) [inline docs/en_US/Release_v1.0.md]: no hyperparameter tuning?

File: docs/en_US/Release_v1.0.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
+
+<p align="center">
+<img src=".././img/release-1-title-1.png" width="100%" />
+</p>
+
+
+From September, 2018 to September, 2019, We are still moving on …
```
Comment (ultmaster) [inline docs/en_US/Release_v1.0.md]: I think September 2018 without comma is more appropriate

File: docs/en_US/Release_v1.0.md (hunk 5 above)
Comment (ultmaster) [inline docs/en_US/Release_v1.0.md]: Should be ";" before besides. Space before `(AutoML)`. Missing space after `Auto-Feature generator & selection,`. Why are some words separated with two spaces?

File: docs/en_US/Release_v1.0.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+</p>
+
+<br />
+<br />
+
+<p align="center">
+<img src=".././img/release-1-title-2.png" width="100%" />
+</p>
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**Step one**：Start with the [Tutorial Document](./Tutorial/Installation.md),and install NNI v1.0 first.<br>
```
Comment (ultmaster) [inline docs/en_US/Release_v1.0.md]: Use `: ` instead of "：". Add space between `,` and `and`.

File: docs/en_US/Release_v1.0.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**Step two**：Find a " Hello world example", follow the [Tutorial Documentation](./Tutorial/QuickStart.md) and have a Quick Start. <br>
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**Step three**：Get familiar with the [WebUI Tutorial](./Tutorial/WebUI.md) and let NNI better assists with your tuning tour.<br>
+
+The fully automated tool greatly improves the efficiency of the tuning process. For more detail about the 1.0 updates, you can refer to [Release 1.0](https://github.com/microsoft/nni/releases) . More of our advance plan, you can refer to our [Roadmap](https://github.com/microsoft/nni/wiki/Roadmap).Besides, we also welcome more and more contributors to join us, there are many ways to participate, please refer to [How to contribute](./Tutorial/Contributing.md
```
Comment (ultmaster) [inline docs/en_US/Release_v1.0.md]: Extra space before `. More of our advance plan`. Missing space after the link of Roadmap.

File: docs/en_US/Release_v1.0.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+
+<br />
+<br />
+
+<p align="center">
+<img src=".././img/release-1-title-2.png" width="100%" />
+</p>
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**Step one**：Start with the [Tutorial Document](./Tutorial/Installation.md),and install NNI v1.0 first.<br>
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**Step two**：Find a " Hello world example", follow the [Tutorial Documentation](./Tutorial/QuickStart.md) and have a Quick Start. <br>
```
Comment (ultmaster) [inline docs/en_US/Release_v1.0.md]: Extra space after `"`. Use `: ` instead of `：`.

File: docs/en_US/Release_v1.0.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+
+ **Great news!**&nbsp;&nbsp;With the tag of **Scalability** and **Ease of Use** , NNI v1.0 is comming. Based on the various types of [Tuning Algorithms](./Tuner/BuiltinTuner.md), NNI has supported the [Neural Architecture Search](docs/en_US/AdvancedFeature/GeneralNasInterfaces.md) and  Auto-Feature generator & selection,which is an exciting news for algorithmic engineers, besides these, NNI v1.0 has made many improvements in the optimization of tuning algorithm,  [WebUI's simplicity and intuition](./Tutorial/WebUI.md) and [platform diversification](./TrainingService/SupportTrainingService.md). NNI has grown into a more intelligent automated machine learning(AutoML) toolkit.
+ 
+
+
+<br/>
+<br/>
+<br/>
+<p align="center">
+<img src=".././img/nni-1.png" width="80%" />
```
Comment (ultmaster) [inline docs/en_US/Release_v1.0.md]: I think there are some "？" in the image, which should be "?"

File: docs/en_US/Release_v1.0.md [hunk 11]
```diff
@@ -0,0 +1,32 @@
+
+<p align="center">
+<img src=".././img/release-1-title-1.png" width="100%" />
+</p>
+
+
+From September, 2018 to September, 2019, We are still moving on …
+
+
+ **Great news!**&nbsp;&nbsp;With the tag of **Scalability** and **Ease of Use**, NNI v1.0 is comming. Based on the various types of [Tuning Algorithms](./Tuner/BuiltinTuner.md), NNI has supported the [Neural Architecture Search](docs/en_US/AdvancedFeature/GeneralNasInterfaces.md) and  Auto-Feature generator & selection, which is an exciting news for algorithmic engineers, besides these, NNI v1.0 has made many improvements in the optimization of tuning algorithm, [WebUI's simplicity and intuition](./Tutorial/WebUI.md) and [platform diversification](./TrainingService/SupportTrainingService.md). NNI has grown into a more intelligent automated machine learning(AutoML) toolkit.
```
Comment (ultmaster) [inline docs/en_US/Release_v1.0.md]: Missing space before `(AutoML)`. Use `;` before `besides these`.

File: docs/en_US/Release_v1.0.md (hunk 5 above)
Comment (rabbit008) [reply docs/en_US/Release_v1.0.md]: fixed

File: docs/en_US/Release_v1.0.md (hunk 6 above)
Comment (rabbit008) [reply docs/en_US/Release_v1.0.md]: ok, fixed~

File: docs/en_US/Release_v1.0.md (hunk 5 above)
Comment (rabbit008) [reply docs/en_US/Release_v1.0.md]: ok ,all fixed. TT

File: docs/en_US/Release_v1.0.md (hunk 7 above)
Comment (rabbit008) [reply docs/en_US/Release_v1.0.md]: ok ,all fixed

File: docs/en_US/Release_v1.0.md (hunk 8 above)
Comment (rabbit008) [reply docs/en_US/Release_v1.0.md]: ok ,all fixed

File: docs/en_US/Release_v1.0.md (hunk 9 above)
Comment (rabbit008) [reply docs/en_US/Release_v1.0.md]: ok ,all fixed

File: docs/en_US/Release_v1.0.md (hunk 11 above)
Comment (rabbit008) [reply docs/en_US/Release_v1.0.md]: ok ,all fixed

File: docs/en_US/Release_v1.0.md (hunk 5 above)
Comment (xuehui1991) [reply docs/en_US/Release_v1.0.md]: I suggest change the “Auto-Feature generator & selection" to "Auto-Feature-Engineering"

Comment (rabbit008) [reply docs/en_US/Release_v1.0.md]: ok, fixed~

**Pull request merged**